delete.php

```diff
@@ -0,0 +1,55 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+use tool_excimer\profile;
+
+/**
+ * Delete profiles.
+ *
+ * @package   tool_excimer
+ * @author    Jason den Dulk <jasondendulk@catalyst-au.net>
+ * @copyright 2021, Catalyst IT
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+require_once(__DIR__ . '/../../../config.php');
+
+require_once($CFG->libdir.'/adminlib.php');
+
+require_admin();
+
+$returnurl = optional_param('returnurl', '', PARAM_LOCALURL);
+if ($returnurl == '') {
+    $returnurl = get_local_referer(false);
+}
```

Comment:
```suggestion
$returnurl = optional_param('returnurl', get_local_referer(false), PARAM_LOCALURL);
```
Would this work instead?

Reply:
Yes. Yes it would.